stan/math/prim/mat/fun/mdivide_left_tri.hpp

```diff
@@ -7,7 +7,11 @@
 #include <stan/math/prim/mat/err/check_multiplicable.hpp>
 #include <stan/math/prim/mat/err/check_square.hpp>
 #ifdef STAN_OPENCL
-#include <stan/math/opencl/opencl.hpp>
+#include <stan/math/opencl/opencl_context.hpp>
+#include <stan/math/opencl/multiply.hpp>
+#include <stan/math/opencl/tri_inverse.hpp>
+#include <stan/math/opencl/transpose.hpp>
+#include <stan/math/opencl/copy.hpp>
 #endif
```

Comment:
I see in cholesky_decompose.hpp you removed all opencl includes, and included one opencl.hpp, any reason we're doing the opposite here?

Reply:
Nope! Thanks for catching that! This is lil' so I'll include it in this PR